client/client.go

```diff
@@ -709,12 +708,90 @@ func (ec *SDKClient) GetGasPrice(
 			newGasPrice.Int(gasPrice)
 		}
 	} else {
-		log.Println("Setting existing gas price")
```

Comment:
removing this because:
* debugging message shouldn't be in prod
* given `GetGasPrice` will be called for every tx construction, we shouldn't add info logging message to for reducing call latency